### PR TITLE
mmctl roles missing from product docs

### PR DESCRIPTION
**Pull request by cwarnermm**
Docs ticket: https://mattermost.atlassian.net/browse/MM-32161

Updated:
 - Self-Managed Admin Guide > Administration > mmctl Command Line Tool (Beta) > mmctl roles
   - Updated mmctl roles documentation based on the mmctl repository

### CODE REVIEW

**Comment by cwarnermm** [comment]
@mgdelacroix In the mmctl repository, the mmctl roles command does not currently include the format or examples. Is this intended?

File: source/administration/mmctl-cli-tool.rst [hunk 1]
```diff
@@ -2976,7 +2976,26 @@ mmctl post list
 mmctl roles
 -----------
 
-This command will be available in a future release.
+**Description**
+
+  Management of user roles.
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:2981]:
```suggestion
  Manage user roles.
```

File: source/administration/mmctl-cli-tool.rst [hunk 2]
```diff
@@ -2976,7 +2976,26 @@ mmctl post list
 mmctl roles
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:2976]: Are there any examples of formatting for this command?

File: source/administration/mmctl-cli-tool.rst (hunk 1 above)
Comment (justinegeffen) [reply source/administration/mmctl-cli-tool.rst:2981]: I couldn't decide whether the original matched the changes you made to the style of the descriptions (which read much better!) or whether it should be changed. LMK which is correct?

**Comment by mgdelacroix** [comment]
> @mgdelacroix In the mmctl repository, the mmctl roles command does not currently include the format or examples. Is this intended?

Indeed, the roles subcommands were missing examples with the new format. I've just sent https://github.com/mattermost/mmctl/pull/312 to add them

File: source/administration/mmctl-cli-tool.rst (hunk 1 above)
Comment (cwarnermm) [reply source/administration/mmctl-cli-tool.rst:2981]: @justinegeffen Your suggestion is a better approach. Thanks!

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Approving for this PR - I'm 3/5 on merging it now and then adding the format for the commands if that PR will take a while. Just my 2c. :)

**Comment by cwarnermm** [comment]
@justinegeffen I believe I have the missing details needed for the documentation, and I can add those today. Might want to hold off merging until the format/example details are in place. Thanks!

**Comment by cwarnermm** [comment]
@justinegeffen I'd like to hear your thoughts on how the formatting & example updates for **mmctl roles** fits in with the mmctl documentation as a whole. My updates are based on https://github.com/mattermost/mmctl/pull/312.

**Review by wiersgallak — APPROVED** [review]

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**